website/sidebars.json

```diff
@@ -170,6 +173,9 @@
           "api/rpc-experimental"
         ]
       }
+    ],
+    "FAQ": [
+      "api/faq"
```

Comment:
i don't think this is what you need for the FAQ file added to validators but i can fix later if you need help with it, no problem.